modules/du/du.go

```diff
@@ -0,0 +1,101 @@
+package du
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+)
+
+// Config is configuration for Du module.
+type Config struct {
+	Paths []string `yaml:"paths"`
+}
+
+// Du is a module for file/folder size monitoring like linux command 'du'
+type Du struct {
+	module.Base // should be embedded by every module
+	Config      `yaml:",inline"`
+
+	charts        *module.Charts
+	collectedDims map[string]bool
+}
+
+func init() {
+	module.Register("du", module.Creator{
+		Defaults: module.Defaults{
+			UpdateEvery:        module.UpdateEvery,
+			AutoDetectionRetry: module.AutoDetectionRetry,
+			Priority:           module.Priority,
+			Disabled:           true,
+		},
+		Create: func() module.Module { return New() },
+	})
+}
+
+func fileSize(path string) (int64, error) {
+	var size int64
+	err := filepath.Walk(path, func(_ string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
```

Comment:
Hello @ski2per ,

This function returns two values according its declaration, but here you are returning only one value, please, fix this.

Reply:
sorry , I missed the fact this is inside another function.

Reply:
Wish it would work, i just jump from python to go 🤣